components/vf-form/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "version": "1.1.2",
-  "name": "@visual-framework/vf-form__core",
-  "description": "vf-form__core component",
+  "version": "2.0.0",
```

Comment:
We should leave the version number "as is" and then  Lerna will update it. Doing like this would force a 2.0.1 release on `lerna publish`

Reply:
Ah - that might have been a clerical error